Examples/UIExplorer/UIExplorerUnitTests/RCTImageLoaderHelpers.m

```diff
@@ -47,9 +47,9 @@ - (BOOL)canLoadImageURL:(NSURL *)requestURL
   return _canLoadImageURLHandler(requestURL);
 }
 
-- (RCTImageLoaderCancellationBlock)loadImageForURL:(NSURL *)imageURL size:(CGSize)size scale:(CGFloat)scale resizeMode:(RCTResizeMode)resizeMode progressHandler:(RCTImageLoaderProgressBlock)progressHandler completionHandler:(RCTImageLoaderCompletionBlock)completionHandler
+- (RCTImageLoaderCancellationBlock)loadImageForURL:(NSURL *)imageURL bundlePath:(NSString *)bundlePath size:(CGSize)size scale:(CGFloat)scale resizeMode:(RCTResizeMode)resizeMode progressHandler:(RCTImageLoaderProgressBlock)progressHandler completionHandler:(RCTImageLoaderCompletionBlock)completionHandler
```

Comment:
I feel bundlePath should be resolved at the point where we create the NSURL. That NSURL should be relative to your custom bundle.